### DISCUSSION TITLE
Add a PWA version for Chrome

### DISCUSSION THREAD

**Discussion by pingshunhuangalex** [monkeytypegame/monkeytype#1018] [Ideas] [answered]
It'd be nice to have a PWA version of the site so it can be installed and launched as an app any time from daily workflow

**Comment by whallin** [comment]
A PWA of Monkeytype would most certainly be awesome to have. And as you say, it's cool to be able to just launch it from your desktop at any moment without having to:
- Open your browser and go to the domain
- Have a shortcut saved on your desktop that opens your browser

A PWA would really make it feel like its own little piece of software on your PC.

**Reply by unrealapex** [reply]
possibly a standalone monkeytype app with Electronjs?.

**Reply by gruvw** [reply]
no need for Electron just make it a PWA with manifest

**Reply by TomzBench** [reply]
electron could allow you to capture control/alt keys - would make the vim lang super awesome to run

**Comment by Miodec** [comment, marked as answer]
I dont really know any ways of making it standalone at the moment and its not high on the priority list either.

**Reply by gruvw** [reply]
Is there any plan to push this higher is the priority list ?  
I really think that this addition could benefit a lot to this (already amazing) project.

(talking about the offline availability)

**Reply by Miodec** [reply]
PWA is already available in Chromium browsers, just click the install icon in the address bar 
<img width="80" alt="image" src="https://github.com/monkeytypegame/monkeytype/assets/13181393/06c93f26-a1ee-46cf-af66-3d47b05e75f9">

**Reply by gruvw** [reply]
Sorry, I edited my comment, I was speaking about the offline availability. Is this feature already implemented ?

**Reply by Miodec** [reply]
Yes

**Comment by gruvw** [comment]
Another advantage that this would bring is offline availability, you could open the app and type in the train.
